website/docs/r/log_search.html.markdown

```diff
@@ -103,12 +102,12 @@ The following arguments are supported:
 ### Schema for `query_parameter`
 - `name` - (Required) The name of the parameter.
 - `description` - (Optional) A description of the parameter.
-- `dataType` - (Required) The data type of the parameter. Supported values are:
+- `data_type` - (Required) The data type of the parameter. Supported values are:
   1. `NUMBER`
   2. `STRING`
   3. `ANY`
   4. `KEYWORD`
-- `value` - (Required) A value for the parameter. Should be compatible with the type set in dataType field.
+- `value` - (Required) The default value for the parameter. Should be compatible with the type set in dataType field.
```

Comment:
"Should be compatible with the type set in dataType field."

-> 

"It should be compatible with the type set in data_type field."